recipe/meta.yaml

```diff
@@ -28,13 +28,21 @@ requirements:
     - numpy >=1.20
     - fasteners
     - numcodecs >=0.10.0
+  run_constrained:
+    # Add `zarr[jupyter]` dependencies to `run_constrained`.
+    # That way the correct versions of these will be installed
+    # when these packages are requested. Otherwise they are not
+    # installed.
+    - notebook
+    - ipytree >=0.2.2
+    - ipywidgets >=8.0.0
 
 test:
   requires:
     - pytest
-    # Needs `pkg_resources` in the tests
-    # xref: https://github.com/zarr-developers/zarr-python/pull/813
-    - setuptools
+    # Needs `packaging` in the tests
+    # xref: https://github.com/zarr-developers/zarr-python/pull/1450
+    - packaging
```

Comment:
Updated the test dependencies to handle the change from PR ( https://github.com/zarr-developers/zarr-python/pull/1450 )